src/DI/AutoRegistratorExtension.php

```diff
@@ -16,14 +16,27 @@ final class AutoRegistratorExtension extends CompilerExtension
 
 	private array $defaults = [
 		'scanDirs' => [],
-		'skipDirs' => [
-			'Exception',
```

Comment:
This is a BC break. Please release a major version or revert this change.

Reply:
Exceptions are now skipped because they are subclass of Throwable